lib/commonConfig.js

```diff
@@ -461,6 +462,11 @@ function processEnvironmentVariables() {
         config.defaultExpressionLanguage =
             process.env.IOTA_DEFAULT_EXPRESSION_LANGUAGE;
     }
+    if(process.env.IOTA_RELAX_TEMPLATE_VALIDATION){
+        config.relaxTemplateValidation = process.env.IOTA_RELAX_TEMPLATE_VALIDATION === 'true';
+    } else {
```

Comment:
I don't think this second clause is necessary - if `config.relaxTemplateValidation` doesn't exist it will be falsy anyway.

Reply:
Hi Jason, yes you're absolutely right, i just didn't want to leave an undefined value for future comparisons.